runtime-sdk/modules/rofl-market/src/test.rs

```diff
@@ -768,6 +768,84 @@ fn test_instance_management() {
     assert_eq!(cmds[1].id, 2.into());
     assert_eq!(&cmds[1].cmd, b"and the last one.");
 
+    // Transfer instance to a new admin and then back again.
```

Comment:
Just mentioning that we should probably avoid having so long tests, for multiple reasons:
- It is simple to verify these changes, but very hard to find latter which part is testing admin transfer functionality.
- One might assume that he needs to know everything that was going on up to this point.
- ...

Reply:
I agree and was wondering about the same thing when adding this test. Will move into a separate test.